leaf-ui/rappid-extensions/AssignmentsTable.js

```diff
@@ -150,16 +150,19 @@ var AssignmentsTable = Backbone.View.extend({
     displayAbsoluteIntentionAssignments: function() {
         this.model.getIntentions().forEach(intentionCell => {
             var intentionBbm = intentionCell.get('intention');
-            var funcSegList = intentionBbm?.get('evolvingFunction')?.get('functionSegList');
+            var evolvingFunction = intentionBbm.get('evolvingFunction');
```

Comment:
So sorry I missed this earlier, I think getIntentions() was updated to return intentionBBMs already so you don't need to get the intentionBbm from the intentionCell

Reply:
Wait, so do we just use intentionCell in place of intentionbbm?